.github/workflows/build.yml

```diff
@@ -124,6 +124,11 @@ jobs:
       - name: Set plugin version
         run: |
           sed -i "s~\${betonquest\.version}~${VERSION}~g" ./src/main/resources/plugin.yml
+      - name: Set pom.xml build number
+        if: "env.VERSION_TYPE == 'developement'"
+        run: |
+          BUILD_NUMBER=$(echo $VERSION | sed 's/.*-DEV-//')
+          mvn versions:set-property -DgenerateBackupPoms=false -Dproperty=betonquest.version.buildnumber -DnewVersion="$BUILD_NUMBER"
```

Comment:
Assuming this is being run on bash we could just use parameter expansion:
```suggestion
          mvn versions:set-property -DgenerateBackupPoms=false -Dproperty=betonquest.version.buildnumber -DnewVersion="${BUILD_NUMBER##*-DEV-}"
```

Reply:
then i also add this to the release script, where i originally copied it from

Reply:
dont you need to use VERSION then instead of BUILD_NUMBER?

Reply:
I adapted this